action.yml

```diff
@@ -99,7 +100,7 @@ runs:
         echo "::set-output name=clean_dockerfile_path::${tmp//./}"
 
     - name: Build image
-      if: steps.check_exclude_file.outputs.files_exists == 'false'
+      if: ${{ steps.check_exclude_file.outputs.files_exists == 'false' && steps.check_dockerfile_file.outputs.files_exists == 'true' }}
```

Comment:
This is not needed; the whole job should fail if the Dockerfile does not exist because the `allow_failure` input is set to true in the file-existence-action[1].

[1] https://github.com/marketplace/actions/file-existence#options-%EF%B8%8F

Reply:
Ok, I will remove it.